app/config/services.yml

```diff
@@ -12,3 +13,25 @@ services:
             - @cmf_core.publish_workflow.checker
         tags:
             - {name: "sonata.block"}
+    cmf_simple_cms.initializer:
+        class: AppBundle\Initializer\HomePageInitializer
+        arguments:
+            - %cmf_simple_cms.persistence.phpcr.basepath%
+            - %app_bundle.document.seo_page%
```

Comment:
i think we should put the class name here directly, not define a parameter for this